src/pages/Schedule.tsx

```diff
@@ -30,7 +19,7 @@ const SUBMIT = gql`
 `;
 
 export const Schedule = () => {
-  const [selectValue, setSelectValue] = useState("");
+  const [viewState, setViewState] = useState("table");
```

Comment:
Again not a blocker but if you wanted to you could use an enum or string union here instead of just `string`

Reply:
updated